backends-clickhouse/src/main/java/io/glutenproject/vectorized/SimpleExpressionEval.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.glutenproject.vectorized;
+
+import java.util.Iterator;
+
+import io.substrait.proto.Plan;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.glutenproject.execution.ColumnarNativeIterator;
+import io.glutenproject.substrait.plan.PlanNode;
+
+public class SimpleExpressionEval implements AutoCloseable, Iterator<Long> {
```

Comment:
How about making this class to be common-used? @zhouyuan Do you think we also need this part?